schemas/context/experienceevent-advertising.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/experienceevent-advertising",
```

Comment:
@kstreeter - I know you do not like this naming convention I like it because it manifests the `"meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"]` in a way that is easier to see. I'm a simple person... Do you have an alternative recommendation?